.github/workflows/performance-tests.yml

```diff
@@ -85,14 +86,24 @@ jobs:
         if: steps.deploy.outcome == 'success' && steps.tests.outcome == 'success'
         run: make -C performance down
 
-      # TODO(danh): get Slack report generated as part of k6 output
-      # - name: Post run report
-      #   if: always()
-      #   id: slack
-      #   uses: slackapi/slack-github-action@v1.24.0
-      #   with:
-      #     # goes to #ci-bots
-      #     channel-id: 'C9LFPNA3Y'
-      #     slack-message: NYI
-      #   env:
-      #     SLACK_BOT_TOKEN: ${{ secrets.PERFORMANCE_SLACK_BOT_TOKEN }}
+      - name: Get Slack report contents
+        if: always()
+        id: slack-report
+        run: |
+          ls performance/reports
+          cat performance/reports/*.results.txt
```

Comment:
When printing several files for debugging, I like to use `more file* | cat` or `grep . file*` so it shows the filenames.  The one with more shows the filename in a header before each file, while grep prefixes each line with the filename, so which one depends on the situation. But it's nice to know which file lines come from. :)

Reply:
My favorite for that is `head -n-0`.